### DISCUSSION TITLE
Syncing PyPi with Git releases

### DISCUSSION THREAD

**Discussion by Raverss** [albumentations-team/albumentations#1878] [Q&A]
I have opened an [issue](https://github.com/albumentations-team/albumentations/issues/1867) that was successfully resolved and added to a [release 1.4.12](https://github.com/albumentations-team/albumentations/compare/1.4.12...main#diff-2f4c1294db48f50c674479baf7ddbf7b6a0829625af5a2088d358a90b2ffa1c7L9). However when I do check PyPi [release 1.4.12](https://pypi.org/project/albumentations/1.4.12/#files) and download manually the files I can see that the file check_version.py still contains code for setting up the logger. Why is PyPi version 1.4.12 different from the Git release 1.4.12?

Is there some kind of delay before the Git release is pushed to the PyPi?

**Comment by ternaus** [comment]
When I add release notes at GitHub, corresponding version is uploaded to PyPI => they are synced.

But! If there was a serious bug in this version, new hot fix version is released and version with the bug gets removed from PyPI => some versions on PyPI are not available for download.
----
But this should not be related to the issue that you face.

Does code for the fix exist in version 1.4.13?

**Reply by Raverss** [reply]
I downloaded manually files from PyPi again this morning from this link: https://pypi.org/project/albumentations/1.4.13/#files and it does not include the fix. The file albumentations-1.4.13/albumentations/check_version.py lines 8 - 10 still contains basicConfig call.

To double check I have also uninstalled albumentations from my Python installation, purged the pip cache, and redownloaded 1.4.13 again. BasicConfig call is there too, exactly the same as in the manually downloaded files.

According to the PyPi, the files are from august 6.

**Comment by ternaus** [comment]
Thanks. Will release new version today and check that fix is there.

**Comment by Raverss** [comment]
### Update
I checked this morning and saw that you pushed 1.4.14 7 hours ago to PyPi. It does have the fix included. Thank you @ternaus :) I'm closing this discussion.

**Comment by ternaus** [comment]
@Raverss Thank you for pointing out the issue.

If you have any issue reports or feature proposals, ping me directly or create and Issue, will try to answer ASAP.
